lib/msal-common/src/crypto/ICrypto.ts

```diff
@@ -35,4 +35,8 @@ export interface ICrypto {
      * Generate PKCE codes for OAuth. See RFC here: https://tools.ietf.org/html/rfc7636
      */
     generatePkceCodes(): Promise<PkceCodes>;
+    /**
+     * Generates an JWK RSA S256 Thumbprint
+     */
+    getPublicKeyThumprint(): Promise<string>;
```

Comment:
Need to change this in a number of places
```suggestion
    getPublicKeyThumbprint(): Promise<string>;
```

Reply:
done